lib/repl.js

```diff
@@ -1480,29 +1478,20 @@ function complete(line, callback) {
   } else if ((match = RegExpPrototypeExec(fsAutoCompleteRE, line)) !== null &&
              this.allowBlockingCompletions) {
     ({ 0: completionGroups, 1: completeOn } = completeFSFunctions(match));
-  // Handle variable member lookup.
-  // We support simple chained expressions like the following (no function
-  // calls, etc.). That is for simplicity and also because we *eval* that
-  // leading expression so for safety (see WARNING above) don't want to
-  // eval function calls.
-  //
-  //   foo.bar<|>     # completions for 'foo' with filter 'bar'
-  //   spam.eggs.<|>  # completions for 'spam.eggs' with filter ''
-  //   foo<|>         # all scope vars with filter 'foo'
-  //   foo.<|>        # completions for 'foo' with filter ''
```

Comment:
Note: I'm removing this comment since I think that it is inaccurate (and as a consequence misleading), since  various type of lines pass through here, like for example `{ a: true }`

I think/hope that the code structure makes it clear enough what this `else if` block is for (I also have half a mind to do some refactoring to also make things clearer later on :thinking:)